src/Mondu/Mondu/Presenters/PaymentInfo.php

```diff
@@ -129,7 +134,7 @@ public function get_mondu_payment_html($pdf=false) {
       <section class="woocommerce-order-details mondu-payment">
         <table>
           <tr>
-            <td><strong><?php _e('Account holder', 'mondu'); ?>:</strong></td>
+            <td><strong><?php _e('Account holder', 'mondu'); ?>:&nbsp;&nbsp;&nbsp;</strong></td>
```

Comment:
**suggestion (non-blocking):** What do you think about formatting these values using CSS class instead of adding non-breaking spaces?

Reply:
Makes sense, I'll try to add a stylesheet instead of non-breaking spaces.